yt_dlp/extractor/litv.py

```diff
@@ -68,24 +60,33 @@ def _real_extract(self, url):
 
         webpage = self._download_webpage(url, video_id)
 
+        if '<meta http-equiv="refresh" content="1;url=https://www.litv.tv/"' in webpage:
+            raise ExtractorError('No such content', expected=True)
```

Comment:
should we use regex to match slight variations in this pattern?

Reply:
I bet this will never change, but yes, I can use a regex if you prefer.